src/foam/nanos/boot/Boot.java

```diff
@@ -94,6 +95,21 @@ public Object create(X x) {
       }
     });
 
+    root_ = root_.putFactory("user", new XFactory() {
+      public Object create(X x) {
+        logger.warning("Deprecated use of x.get(\"user\")");
```

Comment:
```suggestion
        logger.warning(new Exception("Deprecated use of x.get(\"user\")"));
```
So it's easy to track down where it's occurring.